backend/index.js

```diff
@@ -15,4 +18,44 @@ app.post('/book', (req, res) => {
 
 });
 
-app.listen(port, () => console.log(`Hello world app listening on port ${port}!`));
\ No newline at end of file
+app.listen(port, () => console.log(`Hello world app listening on port ${port}!`));
+
+//lista
+app.post('/list/createlist/:iduser/:name', (req,res ) => {
+    iduser: number = req.params.iduser;
+    name:string = req.params.name;
+     newlist: ListInterface={
+        id: number,
+        name:string,
+        cards:Array,
+     }
+
+
+})
+
+app.delete('/list/deleteList/:iduser/:idlist', (req,res) =>{
+
+    iduser: number = req.params.iduser;
+    idlist: number = req.params.idlist;
+    //this.list.remove(idlist)
+})
+
+app.patch('/list/updateList/:iduser/:idlist' ,(req, res) =>{
```

Comment:
- It seems that this endpoint doesn't work
- Is it ok to just replace list's name?
- There is no response or error handling